### PR TITLE
Set up a "Hello world" project with two apps

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
## Below is the description of the PR

- [ ] Create a new Rails API app called 'hello-rails-back-end'.
- [ ] Initialize your project with Git.
- [ ] Make sure that your project has a Postgres database set up.
- [ ] Create a table for storing your messages and create 5 different greetings.
- [ ] Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).
- [ ] In the README file for the Rails API, add a link to your React app.

### CODE REVIEW

File: db/migrate/20220928165335_create_greetings.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateGreetings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :greetings do |t|
+      t.string :greeting
+
+      t.timestamps
+    end
+  end
+end
```
Comment (jerryowusu) [inline db/migrate/20220928165335_create_greetings.rb:9]: According to the project requirements, you are suppose to create 5 different greeting messages after creating the Greeting table. Unfortunately I am not able to see the messages after all the configurations and setups, hence it is showing null at the front end. Kindly consider setting up the project well to fix this issue. Also I saw that you added your database username and password which could also create issues when trying to acess it from a different system. :pray:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Kaweesi-Matia , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost-there-reach


## Highlights
Great effort so far :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220928165335_create_greetings.rb (hunk 1 above)
Comment (Kaweesi-Matia) [reply db/migrate/20220928165335_create_greetings.rb:9]: Created five different messages

File: db/seeds.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
```
Comment (chrystalme) [inline db/seeds.rb:6]: Kindly include a seed file to help create greetings for your application when a reviewer is testing.

You can do this by using `Greeting.create!(greeting: "Hi")` and other greetings

**Review by chrystalme — CHANGES_REQUESTED** [review]
Hi @Kaweesi-Matia , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linters is working and passes ✅ 
- Gitflow adhered to ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Kindly check your README.md file for spelling errors
- Also, check the project to clone name that it matches with your application name and url.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Kaweesi-Matia , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice app connection 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**